src/spoke/interfaces/IVaultSpoke.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: UNLICENSED
+// Copyright (c) 2025 Aave Labs
+pragma solidity ^0.8.0;
+
+import {IHub} from 'src/hub/interfaces/IHub.sol';
+import {IERC4626} from 'src/dependencies/openzeppelin/IERC4626.sol';
+import {IERC2612} from 'src/dependencies/openzeppelin/IERC2612.sol';
+import {EIP712Types} from 'src/libraries/types/EIP712Types.sol';
+
+/// @title IVaultSpoke
+/// @author Aave Labs
+interface IVaultSpoke is IERC4626, IERC2612 {
```

Comment:
reviewer-note: we do not inherit oz's 4626 impl bc we end up overriding every single fn except preview*'s